includes/class-wc-gateway-ppec-client.php

```diff
@@ -458,72 +458,7 @@ protected function _get_details_from_cart() {
 			'items'             => $this->_get_paypal_line_items_from_cart(),
 		);
```

Comment:
since you pulled out the logic of details builder into its own method, maybe we could bring this `$details` initialization too?